### ISSUE TITLE
--help fails

### ISSUE THREAD

**Issue by erikrikarddaniel** [erikrikarddaniel/pfitmap-nextflow#31]
In the current master branch version, running `$ nextflow run ../dev/pfitmap-nextflow/main.nf --help` fails with the following message:

```
N E X T F L O W  ~  version 20.01.0
Launching `../dev/pfitmap-nextflow/main.nf` [hungry_hodgkin] - revision: 0a5a062030
Missing inputgenomes parameter
[Parameter error] Please specify the parameter --inputgenomes
See more using --help

 -- Check script '/home/dl/dev/pfitmap-nextflow/main.nf' at line: 33 or see '.nextflow.log' file for more details
```

I assume checking the param.help comes too late in the script.

**Comment by GhadaNOUAIRIA** [comment]
Fixed!
Description of parameter: inputgenomes changed in the help message

**Comment by erikrikarddaniel** [comment]
When you fix something, make sure you mention the commit that fixes it, i.e. fixed in cee6f0db938941d980e5af1368c9ad06515b0806.
